mix.exs

```diff
@@ -45,6 +45,7 @@ defmodule Kepler.MixProject do
       {:phoenix_live_dashboard, "~> 0.6"},
       {:esbuild, "~> 0.4", runtime: Mix.env() == :dev},
       {:swoosh, "~> 1.3"},
+      {:tailwind, "~> 0.1.6", only: :dev},
```

Comment:
`only: :dev`, intentional?

Reply:
Yes. That is what the docs say. The reason is when you go to production, you actually compile your assets so you don’t need the Elixir module